webpack.config.js

```diff
@@ -38,6 +38,7 @@ const GutenbergBlocksConfig = {
 		'product-top-rated': './assets/js/blocks/product-top-rated/index.js',
 		'products-by-attribute': './assets/js/blocks/products-by-attribute/index.js',
 		'featured-product': './assets/js/blocks/featured-product/index.js',
+		'product-tag': './assets/js/blocks/product-tag/index.js',
```

Comment:
Can be done in a follow-up (or we can create a `good first issue` ticket), but this block should probably be renamed to `product-by-tag` in order to keep it consistent with other blocks.

Reply:
There is a `product-category` which this block follows. There is already inconsistency :(

Reply:
Right, and I see it's not as easy to fix as I thought, because once the blocks have been added to a post, we can't change their name without breaking them. Let's leave it like that for now, then.